int32 xmin
int32 ymin
int32 xmax
int32 ymax
int32 id
int32 center_x
int32 center_y

